CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# Upcoming
+
+* Remove @warn from link helpers because they are annoying
```

Comment:
No, but I think the changelog message should be changed to that to be more descriptive. "Because annoying/rubbish/whatever" doesn't tell us much about the real reason why they were removed =)